scripts/test/SANS/gui_logic/run_tab_presenter_test.py

```diff
@@ -138,7 +138,7 @@ def test_that_will_load_user_file(self):
         # Assert certain function calls
         self.assertTrue(view.get_user_file_path.call_count == 3)
         self.assertTrue(view.get_batch_file_path.call_count == 2)  # called twice for the sub presenter updates (masking table and settings diagnostic tab)  # noqa
-        self.assertTrue(view.get_cell.call_count == 64)
+        self.assertEqual(view.get_cell.call_count, 68)
```

Comment:
why has this increase?

Reply:
This has increased because we have added a new column to the table so as we loop through the table getting values from cells we do so four more times overall.